deploy/docker_compose.dev.yml

```diff
@@ -60,6 +60,29 @@ services:
       redis:
         condition: service_healthy
 
+  celery-flower:
+    build:
+      context: .
+      dockerfile: dev.dockerfile
+      args:
+        - HOST_UID=${HOST_UID}
+        - HOST_GID=${HOST_GID}
+    restart: always
+    entrypoint:
+      - /bin/sh
+      - start_flower.sh
+    ports:
+      - "5555:5555"
+    env_file:
+      - .env
+    volumes:
```

Comment:
If we run `docker-compose down` and then `docker-compose up`, the previous Flower data is lost. We need to use persistent volume to retain the data.